.travis.yml

```diff
@@ -3,36 +3,23 @@ node_js: "8"
 script:
   - yarn lint
   - yarn test
-  - if [ "$TRAVIS_PULL_REQUEST" = "false" ] && ([ "$TRAVIS_BRANCH" == "master" ] || [ "$TRAVIS_BRANCH" == "staging" ]); then yarn run build; fi
+  - if [ "$TRAVIS_PULL_REQUEST" = "false" ] && ([ "$TRAVIS_BRANCH" == "master" ] || [ "$TRAVIS_BRANCH" == "staging" ] || [ "$TRAVIS_BRANCH" == "ep-upgrade" ]); then yarn run build; fi
 after_success:
   - bash <(curl -s https://codecov.io/bash)
 cache:
   yarn: true
   directories:
   - node_modules
-
 deploy:
   - provider: s3
-    access_key_id: $AWS_ACCESS_KEY_ID
-    secret_access_key: $AWS_SECRET_ACCESS_KEY
-    bucket: cru-givestage
+    access_key_id: $8_1_AWS_ACCESS_KEY_ID
+    secret_access_key: $8_1_AWS_SECRET_ACCESS_KEY
+    bucket: cru-give-web-assets-stage
     acl: public_read
     cache_control: "max-age=300"
     local-dir: dist
     skip_cleanup: true
     on:
-      branch: staging
-
-  - provider: s3
-    access_key_id: $AWS_ACCESS_KEY_ID
-    secret_access_key: $AWS_SECRET_ACCESS_KEY
-    bucket: cru-giveprod
-    acl: public_read
-    cache_control: "max-age=3600"
-    local-dir: dist
-    skip_cleanup: true
-    on:
-      branch: master
-
+      branch: ep-upgrade
 after_deploy:
-  - if [ "$TRAVIS_BRANCH" == "master" ]; then aws cloudfront create-invalidation --distribution-id E51L08TW3241I --paths "/*"; fi
+  - if [ "$TRAVIS_BRANCH" == "master" ] || [ "$TRAVIS_BRANCH" == "ep-upgrade" ]; then aws cloudfront create-invalidation --distribution-id E51L08TW3241I --paths "/*"; fi
```

Comment:
This should be two lines, one for `master` and one for `ep-upgrade` with different distribution IDs